### PR TITLE
{bio}[foss/2019a,intel/2019a] Seaborn v0.9.0 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/0d55a3a2f603ebf9270142f0eb3e5605 for a full test report.

File: easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-foss-2019a-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
```
Comment (pescobar) [inline easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-foss-2019a-Python-2.7.15.eb]: being picky ... I like to have an empty line between header and `easyblock` line to improve readability

Comment (boegel) [reply easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-foss-2019a-Python-2.7.15.eb]: That's really pushing it imho, we've never required this (and if we should start doing it, we should make the tests trip over it).

Let's not try and come up with ways to keep PRs open, there's enough of them already ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/412e017d8b30053d5a2b70f05d821e8d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2496.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/70e56ceaf55ad28cddf0683e8422216d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2058.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd34964dd64cab0fb0971405ec8c5a2b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**